doc/guides/maintaining-npm.md

```diff
@@ -17,7 +17,7 @@ changes can be reviewed and landed via the normal consensus seeking process.
 ## Step 1: Clone npm
 
 ```console
-$ git clone https://github.com/npm/npm.git
+$ git clone https://github.com/npm/cli.git
```

Comment:
The line below and at least one other in this file rely on `npm` as a directory name, so maybe this?

```suggestion
$ git clone https://github.com/npm/cli.git npm
```

Reply:
Ah! I just came up with the same. Thank you :)